src/components/AccessibilityBody/index.jsx

```diff
@@ -71,52 +72,7 @@ const AccessibilityBody = (props) => {
           message={messages.a11yBodyExtraInfo}
           tagName="li"
         />
-        <WrappedMessage
```

Comment:
Why were these blocks removed? Were they removed in the updated text we were given?

Reply:
Correct. They were removed in the updated text that is given. 

With the new template of studio-a11y, these text blocks were removed and no longer needed to display. 
See 
https://studio.edx.org/accessibility
vs
<img width="1174" alt="Test" src="https://user-images.githubusercontent.com/4252738/71874016-94aa1b00-3142-11ea-948e-67156c4b1196.png">